doc(release) Release notes for gorgone 24.04.4 (#3837)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx

## Centreon Gorgone

### 24.04.4

Release date: `November 26, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Gorgone] Avoided installation of centreon-engine and centreon-broker on MBI server.

Reviewer: ```suggestion
- [Gorgone] **centreon-engine** and **centreon-broker** are no longer installed on MBI servers.
```
